src/main/java/edu/ucsb/courses/controllers/BasicSearchController.java

```diff
@@ -1,24 +1,27 @@
 package edu.ucsb.courses.controllers;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
-import javax.validation.Valid;
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletResponse;
+
 import com.fasterxml.jackson.core.JsonProcessingException;
+
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
-
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 import edu.ucsb.courses.services.UCSBCurriculumService;
 
+// import edu.ucsb.courses.services.DownloadCsvService;
+
```

Comment:
Is it necessary to keep this commented code in the final

Reply:
^^^ What he said